big-red-shuttle/StopsViewController.swift

```diff
@@ -61,7 +64,33 @@ class StopsViewController: UIViewController, GMSMapViewDelegate {
             marker.map = mapView
         }
     }
+    
+    
+    //drawing
+    func initPolyline() {
+        let stops = getStops()
+        let stopsA = Array(stops[1..<9]) //can only have a max of 8 waypoints (i.e. stops that don't include start and end)
```

Comment:
Google maps only allows eight waypoints (stops that aren't origin or destination), so stopsA has a max of eight